src/openrc-init/openrc-init.c

```diff
@@ -261,14 +262,18 @@ int main(int argc, char **argv)
 	}
 #endif
 
-	printf("OpenRC init version %s starting\n", VERSION);
+	default_runlevel = NULL;
+
+	if (getopt(argc, argv, "q") == 'q')
+				isquiet = true;
+
+	if (!isquiet)
+		printf("OpenRC init version %s starting\n", VERSION);
 
 	if (argc > 1)
-		default_runlevel = argv[1];
-	else
-		default_runlevel = NULL;
+		default_runlevel = argv[optind];
```

Comment:
Keep the `else` part and remove the earlier `default_runlevel = NULL;`. The initialization should happen in one place instead of being split up into two distant places.